src/pages/boundary/api-docs/[[...page]].tsx

```diff
@@ -135,4 +143,5 @@ export async function getStaticProps({ params }) {
 	}
 }
 
+ApiDocsView.layout = ApiDocsLayout
```

Comment:
This file serves as a POC use case, leverage the newly created/exported `PageWithLayout` type.